apps/storefront/src/pages/CompanyOrderList/index.mobile.test.tsx

```diff
@@ -0,0 +1,762 @@
+import {
+  buildB2BFeaturesStateWith,
+  buildCompanyStateWith,
+  builder,
+  buildStoreInfoStateWith,
+  bulk,
+  faker,
+  getUnixTime,
+  graphql,
+  HttpResponse,
+  renderWithProviders,
+  screen,
+  startMockServer,
+  stringContainingAll,
+  userEvent,
+  waitFor,
+  waitForElementToBeRemoved,
+  within,
+} from 'tests/test-utils';
+import { when } from 'vitest-when';
+
+import {
+  CompanyOrderNode,
+  CompanyOrderStatuses,
+  CustomerOrderStatus,
+  GetCompanyOrders,
+} from '@/shared/service/b2b/graphql/orders';
+import { CompanyStatus, CustomerRole, UserTypes } from '@/types';
+
+import MyOrders from '.';
+
+const { server } = startMockServer();
+
+const buildOrderStatusWith = builder<CustomerOrderStatus>(() => ({
+  statusCode: faker.number.int().toString(),
+  systemLabel: faker.word.noun(),
+  customLabel: faker.word.noun(),
+}));
+
+beforeEach(() => {
+  vi.spyOn(document.body, 'clientWidth', 'get').mockReturnValue(500);
+});
+
+const buildCompanyOrderNodeWith = builder<CompanyOrderNode>(() => ({
+  node: {
+    orderId: faker.number.int().toString(),
```

Comment:
Do we care that this can technically return `0` which is an invalid order ID? Same for amounts (would that make orders free?)